incubator/ark-server/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1beta1
```

Comment:
`apps/v1beta2`

We officially support the current and previous minor versions of Kubernetes.